### PR TITLE
feat: Update map for 2.0.3

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: maps/maps.json [hunk 1]
```diff
@@ -159,7 +159,16 @@
...
+    "md5": "dbf19887248b9febdb8766ee64073ebf"
+  },
+  {
+    "name": "Ceralus Farm 1",
+    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/maps/map-ceralus-1.png",
+    "x1": -5120,
+    "z1": -1536,
+    "x2": -3073,
+    "z2": -1025,
+    "md5": "09c9dad8691152716337919575955768"
```
Comment (kristofbolyai) [inline maps/maps.json:171]: I have no idea why this was missing...

File: maps/maps.json [hunk 2]
```diff
@@ -332,6 +332,15 @@
...
   },
+  {
+    "name": "Talespun Citadel",
+    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/maps/map-talespun-citadel.png",
+    "x1": -1536,
+    "z1": 9728,
+    "x2": -1025,
+    "z2": 10751,
+    "md5": "9ad11cb5e69391d1b70e5508c8fc66a8"
+  },
```
Comment (kristofbolyai) [inline maps/maps.json:343]: Moved this so it is in the same place as the Data-Storage output.

File: maps/map-main-3-2.png
Comment (kristofbolyai) [inline maps/map-main-3-2.png:1]: I have to check what happened to the island..

File: maps/map-main-5-2.png
Comment (kristofbolyai) [inline maps/map-main-5-2.png:1]: Chunk error...

**Comment by kristofbolyai** [comment]
Someone didn't commit Island of Fiction mapping data to Data-Storage... Remapped and committed now.

**Review by magicus — APPROVED** [review]

**Pull request merged**